src/control/client/storage.go

```diff
@@ -53,10 +59,16 @@ func (c *connList) StorageScan(p *StorageScanReq) *StorageScanResp {
 // and returns an open stream handle. Receive on stream and send ClientResult
 // over channel for each.
 func StorageFormatRequest(mc Control, parms interface{}, ch chan ClientResult) {
+	// NB: This method is a stub to indicate that it has been replaced
+	// by functionality in the control API, and will be removed when
+	// the client package is removed.
 }
 
 // StorageFormat prepares nonvolatile storage devices attached to each
 // remote server in the connection list for use with DAOS.
 func (c *connList) StorageFormat(reformat bool) StorageFormatResults {
+	// NB: This method is a stub to indicate that it has been replaced
+	// by functionality in the control API, and will be removed when
+	// the client package is removed.
```

Comment:
Actually, looking at this again, is there a reason we can't prune these methods out of the client interface and then remove the implementations? Something else depending on them besides that?

Reply:
... No, actually. I was thinking there was some dependency on the protobuf client definitions, but it looks like I can just delete those. I agree that's better than stubbing things.